Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN rm -rf /etc/service/sshd /etc/my_init.d/00_regen_ssh_host_keys.sh
 
 # Dependencies
 RUN DEBIAN_FRONTEND='noninteractive' \
-  apt-get -y --force-yes install bash apt-utils python-pip python-dev build-essential git software-properties-common python-software-properties libpq-dev g++ make libpcre3 libpcre3-dev \
+  apt-get update && apt-get -y --force-yes install bash apt-utils python-pip python-dev build-essential git software-properties-common python-software-properties libpq-dev g++ make libpcre3 libpcre3-dev \
```

Comment:
A couple things from the [Docker best-practices page](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers):
- combine the packages lists from the two `apt-get install` commands
- for legibility, sort them and put them into a list with one package per line
- clear out the apt cache as part of the same RUN, as each RUN creates a layer (see the [example in CT's Dockerfile](https://github.com/ministryofjustice/correspondence_tool_staff/blob/develop/Dockerfile.base#L13))